### ISSUE TITLE
20gig issue with latest Windows 10/11

### ISSUE THREAD

**Issue by MrJonii** [adamrehn/ue4-docker#282]
Output of the `ue4-docker info` command:

```
ue4-docker version:         0.0.102 (latest available version is 0.0.102)
Operating system:           Windows 10 Pro (Build 22000.1219)
Docker daemon version:      20.10.21
NVIDIA Docker supported:    No
Maximum image size:         800GB
Available disk space:       847.03 GiB
Total system memory:        63.93 GiB physical, 9.5 GiB virtual
CPU:                        12 physical, 24 logical (AMD64 Family 25 Model 33 Stepping 0, AuthenticAMD)
```
```
ue4-docker version:         0.0.102 (latest available version is 0.0.102)
Operating system:           Windows 10 Pro (Build 19045.2251)
Docker daemon version:      20.10.21
NVIDIA Docker supported:    No
Maximum image size:         800GB
Available disk space:       1.03 TiB
Total system memory:        31.93 GiB physical, 8.44 GiB virtual
CPU:                        16 physical, 32 logical (Intel64 Family 6 Model 45 Stepping 7, GenuineIntel)
```

Additional details:

- Are you accessing the network through a proxy server? No

Failing ue4-docker diagnostics 20gig on 2 different machines, the first one running on Windows 11 Pro, the second on Windows 10 Pro. I know there was a known issue with some versions of Windows not being able to build images larger than 20gb, but those 2 don't seem to belong to those. Tried in both HyperV and WSL2 isolations.

`ue4-docker diagnostics 20gig` output:

```
[ue4-docker diagnostics] Running diagnostic: "Check for Docker 20GiB COPY bug"
[ue4-docker diagnostics] [20gig] Attempting to COPY more than 20GiB between layers...
[ue4-docker diagnostics] [20gig] Run: ['docker', 'build', '-t', 'adamrehn/ue4-docker/diagnostics:20gig', 'D:\\python\\lib\\site-packages\\ue4docker\\dockerfiles\\diagnostics\\20gig\\windows', '--build-arg', 'BASETAG=ltsc2022', '--isolation=hyperv', '-m', '4GiB']
...
Step 10/10 : COPY --from=intermediate C:\stuff C:\stuff
failed to copy files: failed to copy directory: write \\?\Volume{14d77665-70d5-4618-a741-27e6a2637fbd}\stuff\bigfile_3.txt: There is not enough space on the disk.
[ue4-docker diagnostics] [20gig] Build failed!
[ue4-docker diagnostics] [20gig] Cleaning up...
[ue4-docker diagnostics] [20gig] Run: ['docker', 'system', 'prune', '-f']
Deleted Networks:
WSL
Default Switch

Deleted Images:
deleted: sha256:210998bcc0e5ce7c81e1301ecd9d402557f7c0465991598adc18a4f7101fb0ca
deleted: sha256:bb3121a3dba21350beb3541900db1e224193b80a45a6ebf495cd1b0e0d4a2cca
deleted: sha256:5f81ccac34ded51eaf94529db40ceff5f141733e70b1d81c7730b73b9bcf3178
deleted: sha256:bcc94fbd414e3643e065ace487bd3724854fc2a60eba74d89aa4ddd009fba81c
deleted: sha256:2868be8119b4e5cc15e70fb5fc8fa025b6ea675bceacb2f69db2e12ab025f538
deleted: sha256:0c1a72d4f32680fddfc853e6c9af24bcdc94c1897300c5bff348c0b50e7ece8b
deleted: sha256:0928ffa4835759c6c242aa90a9a023db3673c1538fe3beeff0813a4cb13ea3e4
deleted: sha256:be1c736e5b7a49673284e5249b7b9c8bb0091a62e892057e965b73d76a12c83d
deleted: sha256:ac7a1eb173467796161fef52b3c118733faeb41acfca292f35de60d4764b56fc
deleted: sha256:e7d80b0b615a16dc6531cff6170ebc99a8a79f314f46d4597db8c1383ed53e4e

Total reclaimed space: 22.57GB

[ue4-docker diagnostics] [20gig] Diagnostic failed! The Docker daemon cannot COPY more than 20GiB between layers.

[ue4-docker diagnostics] Diagnostic result: failed
```
```
[ue4-docker diagnostics] Running diagnostic: "Check for Docker 20GiB COPY bug"
[ue4-docker diagnostics] [20gig] Attempting to COPY more than 20GiB between layers...
[ue4-docker diagnostics] [20gig] Run: ['docker', 'build', '-t', 'adamrehn/ue4-docker/diagnostics:20gig', 'c:\\users\\ffsbu\\appdata\\local\\programs\\python\\python37\\lib\\site-packages\\ue4docker\\dockerfiles\\diagnostics\\20gig\\windows', '--build-arg', 'BASETAG=ltsc2019', '--isolation=hyperv', '-m', '4GiB']
...
Step 10/10 : COPY --from=intermediate C:\stuff C:\stuff
failed to copy files: failed to copy directory: write \\?\Volume{12261331-d2bc-45a9-b105-fed4b68e4cd4}\stuff\bigfile_3.txt: There is not enough space on the disk.
[ue4-docker diagnostics] [20gig] Build failed!
[ue4-docker diagnostics] [20gig] Cleaning up...
[ue4-docker diagnostics] [20gig] Run: ['docker', 'system', 'prune', '-f']
Deleted Networks:
Default Switch

Deleted Images:
deleted: sha256:463e65cfae6b2e8582890f840aca46cebe350aadc6883c2068d2ada52c4f6212
deleted: sha256:e89d80de091d4f878d19115009d98f22fc76ac2d17d117f1c802f6378e9ca5c2
deleted: sha256:6f7e73e8939cad3a7783164e258c945cb29eba7ca573e875d2c05a648bc85a60
deleted: sha256:43e255d8ba1cec436b7911d8aede3b619e60e6c3df22cba883f6b7408892ada6
deleted: sha256:471b839485df16c957b526fc0389312b82f49ad27b76c82eb01b458db97ebd85
deleted: sha256:8ae00c058dfb8ce6ae13ba7cd9a280d580b563832aa4bbae5bde2be305e3164c
deleted: sha256:4b8add8c9c59f6e76092ebed68fed357abad618f97f972190c0e0a66dbc6c670
deleted: sha256:6dcf85e8805d23ced587cc04f8b0fe9b3e25fff821015bbdbd1015f23c2b6f10
deleted: sha256:0175401ae61795548c3d49211ea014032d90c7af39242bce89950c87131cbad9
deleted: sha256:5ed4827b275b41029c4da39695af960088dc0267894c1ba3815f97d36f74cd0b

Total reclaimed space: 22.56GB

[ue4-docker diagnostics] [20gig] Diagnostic failed! The Docker daemon cannot COPY more than 20GiB between layers.

[ue4-docker diagnostics] Diagnostic result: failed
```

**Comment by TBBle** [comment]
This is a known Docker issue, irrespective of the version of Windows you're on. See https://github.com/adamrehn/ue4-docker/issues/99#issuecomment-716965829 for reference. It's fixed in the next release of Docker Engine (v22.06, still in beta), so you can either upgrade to the beta version, e.g. if you're using Docker Desktop, one of the current tech previews might have it, or you could switch from Docker Desktop to https://github.com/slonopotamus/stevedore which has backported the relevant fix.

**Comment by TBBle** [comment]
Closing as duplicate, see https://github.com/adamrehn/ue4-docker/issues/99#issuecomment-1079702817 for the current resolution status.
